app/src/ui/repositories-list/repository-list-item.tsx

```diff
@@ -220,9 +200,8 @@ const renderAheadBehindIndicator = (aheadBehind: IAheadBehind) => {
 }
 
 const renderChangesIndicator = () => {
-  const classNames = enableGroupRepositoriesByOwner()
-    ? 'change-indicator-wrapper group-repositories-by-owner'
-    : 'change-indicator-wrapper'
+  const classNames = 'change-indicator-wrapper group-repositories-by-owner'
```

Comment:
I think we should take this opportunity to delete the `.change-indicator-wrapper:not(.group-repositories-by-owner)` meta selector from `_repository_list.scss` since that's never going to be applied now. If you want to tackle that in a separate PR that's fine but I think that a part of removing the feature flags should be removing these feature flag specific css classes.